### PR TITLE
Pkb wordcount beam improvements

### PR DESCRIPTION
**Pull request by asaksena**

### CODE REVIEW

File: perfkitbenchmarker/dpb_service.py [hunk 1]
```diff
@@ -30,6 +30,10 @@
                     'If set, the name of the pre created dpb implementation,'
                     'assumed to be ready.')
 flags.DEFINE_string('dpb_log_level', 'INFO', 'Manipulate service log level')
+flags.DEFINE_string('dpb_job_jarfile', None,
+                    'Executable Jarfile containing workload implementation')
+flags.DEFINE_string('dpb_job_classname', None, 'Classname to be used')
```
Comment (s-deitz) [inline perfkitbenchmarker/dpb_service.py]: Is this a class in the jar file? Add context.

File: perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py [hunk 2]
```diff
@@ -24,13 +24,12 @@
 
 dpb_wordcount_out_base: The output directory to capture the word count results
 
-For dataflow jobs, please build the dpb_dataflow_jar based on
+For dataflow jobs, please build the dpb_job_jarfile based on
 https://cloud.google.com/dataflow/docs/quickstarts/quickstart-java-maven
```
Comment (s-deitz) [inline perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py]: Is this a manual step that has to be done before running this benchmark?

File: perfkitbenchmarker/dpb_service.py (hunk 1 above)
Comment (asaksena) [reply perfkitbenchmarker/dpb_service.py]: Done. Provided additional documentation for the flag.

File: perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py (hunk 2 above)
Comment (asaksena) [reply perfkitbenchmarker/linux_benchmarks/dpb_wordcount_benchmark.py]: Yes. This is a manual step, till the Dataflow SDK/Beam team can provide prebuilt job jars, similar to the artifacts that ship with Spark/Hadoop releases.

**Pull request merged**